src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,201 @@
+    </li>
+    <ul>
+      <li>
+        <a href="https://github.com/blackbaud/skyux-indicators/blob/master/src/app/visual/alert/index.html">https://github.com/blackbaud/skyux-indicators/blob/master/src/app/visual/alert/index.html</a>
+      </li>
+      <li>
+        <a href="https://github.com/blackbaud/skyux-indicators/blob/master/src/app/visual/alert/alert-visual.component.ts">https://github.com/blackbaud/skyux-indicators/blob/master/src/app/visual/alert/alert-visual.component.ts</a>
+      </li>
+      <li>
+        <a href="https://github.com/blackbaud/skyux-indicators/blob/master/src/app/visual/alert/alert-visual.component.html">https://github.com/blackbaud/skyux-indicators/blob/master/src/app/visual/alert/alert-visual.component.html</a>
+      </li>
+      <li>
+        <a href="https://github.com/blackbaud/skyux-indicators/blob/master/e2e/alert.e2e-spec.ts">https://github.com/blackbaud/skyux-indicators/blob/master/e2e/alert.e2e-spec.ts</a>
+      </li>
+    </ul>
+    <li>
+      <b>All new components and changes to existing components must have passing <a stacheRouterLink="/learn/accessibility/test">accessibility tests</a>.</b> This includes automated tests to run during visual regression tests as well as code review and manual keyboard testing.
+    </li>
+    <li>
+      <b>All new components must include demos to accompany the reference documentation in the <a stacheRouterLink="/components">Components section</a> of the SKY UX website.</b> Demos must include all or most features available to the components and examples of all inputs and outputs. In the <a href="https://github.com/blackbaud/skyux2"></a>SKY UX repo, you create subfolders for your components in the <stache-code>src/demos</stache-code> folder and add TypeScript and HTML files for the demos. You create <stache-code>{{ '{' }}componentName}-demo.component.html</stache-code> files to provide HTML templates for demos, <stache-code>{{ '{' }}componentName}-demo.component.ts</stache-code> files to handle the logic, and <stache-code>index.ts</stache-code> files to export demos. For examples of existing demos, serve the the <a href="https://github.com/blackbaud/skyux2">SKY UX repo</a> or view the <a stacheRouterLink="/components">components documentation</a>. We recommend that you also <a stacheRouterLink="." fragment="write-documentation">create the reference documentation for your components</a> in a separate pull request in the <a href="https://github.com/blackbaud/skyux2-docs">SKY UX docs repo</a>.
```

Comment:
I'd raised some questions on this paragraph that Bobby and Steve never responded to. (Maybe because that conversation became outdated?) ... Anyway, this paragraph is inaccurate because we don't want demos created in the SKY UX 2 repo. We currently have an `a` tag before "SKY UX repo" that doesn't do anything because it doesn't wrap any text. Let's change "SKY UX repo" to "SKY UX docs repo," wrap that text with the `a` tag, update the URL to link to the docs repo instead of the skyux2 repo, and delete "in the SKY UX docs repo" at the end of the paragraph. We may need to alter these instructions at some point, but this is the process we are currently following. ... Please just let me know if any of these changes are not clear.